packages/util-package-extender/lib/js/_utils/__mocks__/_get-local-file-list.js

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Why is this mock file in the main bit of code, and not under `__mocks__` somewhere in the `__tests__` directory?

Reply:
This needs to live in a `__mocks__` folder next to where the file is defined, so

```
lib/js/_utils/_get-local-file-list.js
lib/js/_utils/__mocks__/_get-local-file-list.js
```

Or at least that is how Jest used to function?